package.json

```diff
@@ -8,8 +8,8 @@
   "repository": "Polymer/pwa-starter-kit",
   "scripts": {
     "start": "polymer serve",
-    "build": "npm run build:prpl-server && npm run build:static",
-    "build:prpl-server": "polymer build --auto-base-path && rm -rf server/build && mv build server/ && renamer --find node_modules --replace node_assets 'server/build/**' && replace node_modules node_assets server/build -r -s",
+    "build": "gulp build",
```

Comment:
I'd suggest to keep `build` like it was before (`"build": "npm run build:prpl-server && npm run build:static"`) and the `gulpfile.js` will be just for `prpl-server` stuffs.

Btw, thanks for working on the PR @Dabolus!